### ISSUE TITLE
Capex->operations workflow example

### ISSUE THREAD

**Issue by claytonpbarrows** [G-PST/power-flow-exercise#7]
- [x] Upload MAT to github
- [x] MAT -> pandapower -> PyPSA
- [x] Build PyPSA expansion simulation
- [x] Share example PyPSA output

**Comment by rbolgaryn** [comment]
Hi @claytonpbarrows @kdheepak ,

What is the status of the mat file?

Roman

**Comment by claytonpbarrows** [comment]
@rbolgaryn added here: 77fed3a9251f3e0133d78bb5c6078e0d24fac3dc 
sorry for the delay

**Comment by rbolgaryn** [comment]
Thank you @claytonpbarrows !

**Comment by rbolgaryn** [comment]
I am getting deviations of voltage magnitude of -0.056...0.063. This is not satisfactory for the validation of results. We should clarify where the errors are coming from.

**Comment by rbolgaryn** [comment]
In the example_pandapower folder, I added a function to read the .mpc file and return the PyPSA network.

I think it should be checked if all assumptions I made are correct, first of all dropping the not in_service sgens.

Also, shunt elements are not imported to PyPSA. Should we:

1) remove shunts from the original grid?
2) extend the capability of PyPSA to add shunts while converting from pandapower? (preferred)
